### DISCUSSION TITLE
Can you make fields mandatory?

### DISCUSSION THREAD

**Discussion by ovizii** [docusealco/docuseal#73] [Q&A] [answered]
I'm just going through my first tests with a self-hosted docuseal and I added a date and a signature field to a PDF and I was simply able to ignore the signature, fill in the date field and submit. 

Unless I am making a mistake somewhere, this does not seem to make sense to me.

**Comment by ovizii** [comment]
After more tests, it seems a date field is mandatory while a signature field isn't. Is this on purpose?

**Comment by maaper** [comment]
Even I mark fields as mandatory the system allows the user to bypass manually the filling of some info and send the form. 
There is something that is not working fine with that.

**Comment by ovizii** [comment]
Oh, I see you can edit fields and check "required" but that seems to be the default and yet it does not seem to be enforced as I initially said.

**Reply by maaper** [reply]
Indeed no. In some situations, the system override the instruction and let the user send the form without filling all mandatory fields. There should be some small 🐞 bug in some part. 
Probably they will fix in future versions... let's wait for developers to say something on that. @AlexandrToorchyn :)

**Comment by AlexBTurchyn** [comment, marked as answer]
@ovizii You're right. A simple dot as a signature can pass validation, which is a known issue 😞. For now, we rely on users to adhere to the proper signing process .

**Reply by ovizii** [reply]
Thanks for looking into this. I did not type in anything into the signature field, I simply clicked next, filled in the date and clicked submit. 

Thanks for clarifying though.
